hazelcast/src/main/java/com/hazelcast/internal/networking/nonblocking/NonBlockingIOThreadingModel.java

```diff
@@ -97,6 +103,11 @@ public NonBlockingIOThreadingModel(
     private SelectorMode getSelectorMode() {
         if (selectorMode == null) {
             selectorMode = SelectorMode.getConfiguredValue();
+
+            String selectorModeString = SelectorMode.getConfiguredString();
+            if (selectorModeString.startsWith("selectnow,")) {
```

Comment:
Can't we move this string magic to `SelectorMode` instead of spreading this across the code?